engine/profiler/src/profile_render.cpp

```diff
@@ -799,6 +799,8 @@ namespace dmProfileRender
 
     void DeleteProfilerFrame(ProfilerFrame* frame)
     {
+        if (!frame)
```

Comment:
Founnd an issue when shutting down the engine, after the profiler wasn't started properly.